internal/customtypes/config_types.go

```diff
@@ -0,0 +1,9 @@
+package customtypes
+
+type Environment string
+
+const (
+	Development Environment = "development"
```

Comment:
I'd suggest renaming to `EnvDev/EnvProd/EnvTest= ...` so when you import it's much easier to use and reason:

```go
import github.com/Vaserty/go-authorization/internal/customtypes

x := Development
```
vs 
```go
import github.com/Vaserty/go-authorization/internal/customtypes

x := EnvDev
```

Reply:
Yes, You're right,